eng/tools/HelixTestRunner/TestRunner.cs

```diff
@@ -22,33 +22,35 @@ public bool SetupEnvironment()
             EnvironmentVariables.Add("PATH", Options.Path);
             EnvironmentVariables.Add("helix", Options.HelixQueue);
 
-            Console.WriteLine($"Current Directory: {Options.HELIX_WORKITEM_ROOT}");
+            ProcessUtil.PrintMessage($"Current Directory: {Options.HELIX_WORKITEM_ROOT}");
             var helixDir = Options.HELIX_WORKITEM_ROOT;
-            Console.WriteLine($"Setting HELIX_DIR: {helixDir}");
+            ProcessUtil.PrintMessage($"Setting HELIX_DIR: {helixDir}");
             EnvironmentVariables.Add("HELIX_DIR", helixDir);
             EnvironmentVariables.Add("NUGET_FALLBACK_PACKAGES", helixDir);
             var nugetRestore = Path.Combine(helixDir, "nugetRestore");
             EnvironmentVariables.Add("NUGET_RESTORE", nugetRestore);
             var dotnetEFFullPath = Path.Combine(nugetRestore, helixDir, "dotnet-ef.exe");
-            Console.WriteLine($"Set DotNetEfFullPath: {dotnetEFFullPath}");
+            ProcessUtil.PrintMessage($"Set DotNetEfFullPath: {dotnetEFFullPath}");
             EnvironmentVariables.Add("DotNetEfFullPath", dotnetEFFullPath);
             var dumpPath = Environment.GetEnvironmentVariable("HELIX_DUMP_FOLDER");
-            Console.WriteLine($"Set VSTEST_DUMP_PATH: {dumpPath}");
+            ProcessUtil.PrintMessage($"Set VSTEST_DUMP_PATH: {dumpPath}");
             EnvironmentVariables.Add("VSTEST_DUMP_PATH", dumpPath);
+            EnvironmentVariables.Add("DOTNET_CLI_VSTEST_TRACE", "1");
+            EnvironmentVariables.Add("VSTEST_ DISABLE_FASTER_JSON_SERIALIZATION=", "1");
```

Comment:
Is this option doing anything? There is a space in the middle of it and it includes a `=` at the end which isn't normal

Reply:
lol, hmm, I guess not, I'll remove